package.json

```diff
@@ -37,7 +37,7 @@
     "get-stream": "^4.0.0",
     "git-log-parser": "^1.2.0",
     "nyc": "^13.1.0",
-    "semantic-release": "^15.0.0",
+    "semantic-release": "^16.0.0-beta.2",
```

Comment:
No need to update the dependency here, please keep `^15.0.0`

Reply:
👍